### PR TITLE
CI: ClusterFuzzLite integration

### PR DESCRIPTION
**Pull request by DavidKorczynski**

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
As of this writing, the CI correctly reports an expected crash:
```
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 1337
INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes
INFO: A corpus is not provided, starting from an empty corpus
#2	INITED cov: 16 ft: 16 corp: 1/1b exec/s: 0 rss: 50Mb
#3	NEW    cov: 16 ft: 17 corp: 2/2b lim: 4 exec/s: 0 rss: 50Mb L: 1/1 MS: 1 ChangeBit-
 === Uncaught Python exception: ===
CalltreeError: Decoding error when reading CFG file
Traceback (most recent call last):
  File "fuzz_cfg_load.py", line 34, in TestOneInput
  File "/tmp/_MEIu963Q8/fuzz_introspector/cfg_load.py", line 111, in data_file_read_calltree
    raise CalltreeError("Decoding error when reading CFG file")
```

Will push a fix and see if things pass

**Comment by DavidKorczynski** [comment]
@jonathanmetzman I'm having some troubles integrating a Python project with ClusterFuzzLite. The problem occurs in the check_build where it seems like ClusterFuzzLite treats it as a c++ project rather than a Python project.

From the CI logs:
```
 2022-07-22 14:17:27,813 - root - INFO - repo_dir: /github/workspace/storage/fuzz-introspector.
2022-07-22T14:17:27.8198549Z Successfully tagged external-cfl-project-28788aa53f0746e4b6adfe2b8b2060a9:latest
2022-07-22T14:17:27.8462543Z 2022-07-22 14:17:27,845 - root - INFO - Docker container: db2d4fe59c79.
2022-07-22T14:17:27.8475010Z 2022-07-22 14:17:27,845 - root - INFO - Building with address sanitizer.
2022-07-22T14:17:27.8476739Z 2022-07-22 14:17:27,846 - root - INFO - Running: docker run --rm --privileged -e FUZZING_ENGINE=libfuzzer -e CIFUZZ=True -e SANITIZER=address -e ARCHITECTURE=x86_64 -e FUZZING_LANGUAGE=c++ -e OUT=/github/workspace/build-out --volumes-from db2d4fe59c79 external-cfl-project-28788aa53f0746e4b6adfe2b8b2060a9 /bin/bash -c compile.
2022-07-22T14:17:28.1717068Z ---------------------------------------------------------------
2022-07-22T14:17:28.1725688Z Compiling libFuzzer to /usr/lib/libFuzzingEngine.a...  done.
2022-07-22T14:17:28.1817184Z ---------------------------------------------------------------
2022-07-22T14:17:28.1817540Z CC=clang
...
...
...
2022-07-22T14:19:36.9547035Z 2022-07-22 14:19:36,953 - root - ERROR - Request to https://api.github.com/repos/ossf/fuzz-introspector/actions/artifacts?per_page=100&page=1 failed. Code: 401. Response: {'message': 'Bad credentials', 'documentation_url': 'https://docs.github.com/rest'}
2022-07-22T14:19:36.9554093Z 2022-07-22 14:19:36,954 - root - ERROR - Could not get coverage: Github API request failed..
2022-07-22T14:19:36.9554592Z 2022-07-22 14:19:36,954 - root - ERROR - Could not find latest coverage report.
2022-07-22T14:26:56.4222672Z 2022-07-22 14:26:56,421 - root - ERROR - Build check failed.
2022-07-22T14:26:56.4223353Z Build check: stdout: INFO: performing bad build checks for /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load
2022-07-22T14:26:56.4244895Z 2022-07-22 14:26:56,421 - root - ERROR - Error building fuzzers for (commit: 52313a708e6d8f2bec8e768b09385826434a21e2, pr_ref: refs/pull/410/merge).
2022-07-22T14:26:56.4245603Z INFO: performing bad build checks for /tmp/not-out/tmpcsnqifqm/fuzz_report_generation
2022-07-22T14:26:56.4246021Z Retrying failed fuzz targets sequentially 2
2022-07-22T14:26:56.4246533Z INFO: performing bad build checks for /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load
2022-07-22T14:26:56.4247102Z INFO: performing bad build checks for /tmp/not-out/tmpcsnqifqm/fuzz_report_generation
2022-07-22T14:26:56.4247465Z Broken fuzz targets 2
2022-07-22T14:26:56.4249691Z ('/tmp/not-out/tmpcsnqifqm/fuzz_cfg_load', CompletedProcess(args=['bad_build_check', '/tmp/not-out/tmpcsnqifqm/fuzz_cfg_load'], returncode=2, stdout=b'BAD BUILD /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load is not built for architecture: x86_64\nfile command output: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: POSIX shell script, ASCII text executable\ncheck_mixed_sanitizers test will fail.\nBAD BUILD: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load does not seem to be compiled with ASan.\n', stderr=b'objdump: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: file format not recognized\nobjdump: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: file format not recognized\nobjdump: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: file format not recognized\nobjdump: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: file format not recognized\n'))
2022-07-22T14:26:56.4251675Z BAD BUILD /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load is not built for architecture: x86_64
2022-07-22T14:26:56.4252320Z file command output: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: POSIX shell script, ASCII text executable
2022-07-22T14:26:56.4252772Z check_mixed_sanitizers test will fail.
2022-07-22T14:26:56.4253281Z BAD BUILD: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load does not seem to be compiled with ASan.
2022-07-22T14:26:56.4253849Z objdump: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: file format not recognized
2022-07-22T14:26:56.4254391Z objdump: /tmp/not-out/tmpcsnqifqm/fuzz_cfg_load: file format not recognized
```

Because `FUZZING_LANGUAGE` is set to `c++` the bad build check does not return here: https://github.com/google/oss-fuzz/blob/f014b858d9f05ac9a8fb50ad6a54b0b3819dcfe8/infra/base-images/base-runner/bad_build_check#L304-L307 but rather continues.

Am struggling to find where I go wrong, could you assist?

File: .github/workflows/cflite_pr.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      fail-fast: false
+      matrix:
+        sanitizer: [address]
+    steps:
+    - name: Build Fuzzers (${{ matrix.sanitizer }})
+      id: build
+      uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+      with:
+        sanitizer: ${{ matrix.sanitizer }}
+        language: python
```
Comment (jonathanmetzman) [inline .github/workflows/cflite_pr.yml:21]: I think you needed to add this.
It's confusing that you need to specify language twice

Comment (DavidKorczynski) [reply .github/workflows/cflite_pr.yml:21]: awesome, thanks! The doubling is in essence fine for my part but it should probably be documented here https://google.github.io/clusterfuzzlite//build-integration/python-lang/

**Review by AdamKorcz — APPROVED** [review]

**Pull request merged**